weave-js/src/components/Panel2/PanelGroup.tsx

```diff
@@ -34,7 +49,7 @@ import {
 } from './ChildPanel';
 import * as ConfigPanel from './ConfigPanel';
 import {IconAddNew as IconAddNewUnstyled} from './Icons';
-import {LayoutSections} from './LayoutSections';
+// import {LayoutSections} from './LayoutSections';
```

Comment:
Unless this is very temporary I tend to prefer removing entirely instead of just commenting out.

Reply:
Not sure yet.